packages/smooth_app/lib/data_models/continuous_scan_model.dart

```diff
@@ -40,6 +40,8 @@ class ContinuousScanModel with ChangeNotifier {
   final String languageCode;
   final String countryCode;
 
+  QRViewController? qrViewController;
```

Comment:
In dart there is no way to check if a late variable is initialized or not, so I am always a bit worried about to use them when it's unclear if it is. With the current behaviour it shouldn't happen since it doesn't try to restart the camera if it has not yet been build but it doesn't have to be alwys the case.
Made it private 👍🏼